hydra-node/json-schemas/logs.yaml

```diff
@@ -516,6 +516,25 @@ definitions:
               The Party emitting the log entry.
           effect:
             $ref: "#/definitions/Effect"
+      - title: LogicOutcome
+        description: >-
+          Outcome produced, to transition the Head protocol, after processing an input event.
```

Comment:
Not all outcomes do result in transitions. In fact, only the `NewState` one is changing the head state. An outcome is rather a decision the node took given some event.

Reply:
rewriting